src/main/java/com/company/Main.java

```diff
@@ -1,32 +1,134 @@
 package com.company;
 
+import com.sun.source.tree.SwitchTree;
+
+import java.util.Arrays;
 import java.util.Formatter;
 import java.util.Scanner;
 
 public class Main {
 
     public static void main(String[] args) {
-        double result = 0;
         Scanner in = new Scanner(System.in);
-        System.out.println("Pls enter first value");
-        double value1 = in.nextInt();
-        System.out.println("Pls enter second value");
-        double value2 = in.nextInt();
-        System.out.println("Pls enter operator sum(+),mult(*),sub(-) or div(/)");
-        String operator = in.next();
-
-        if (operator.equalsIgnoreCase("sum") || operator.equals("+"))
-            result = value1 + value2;
-        else if (operator.equalsIgnoreCase("mult") || operator.equals("*"))
-            result = value1 * value2;
-        else if (operator.equalsIgnoreCase("sub") || operator.equals("-"))
-            result = value1 - value2;
-        else if (operator.equalsIgnoreCase("div") || operator.equals("/"))
-            result = value1 / value2;
-        else System.out.println("Wrong value, pls try again");
-
-        System.out.println("Your result  " + String.format("%.2f%n", result));
+        System.out.println("Pls enter initial value of 1st array");
+        var inValueForFirstArray = in.nextInt();
+        System.out.println("Pls enter size of arrays");
+        var sizeOfArrays = in.nextInt();
+        System.out.println("Pls enter step of 1st array");
+        var stepForFirstArray = in.nextInt();
+        System.out.println("Pls enter increment operator of 1st array (+),(*),(-) or (/)");
+        String operatorForFirstArray = in.next();
+        System.out.println("Pls enter initial value of 2nd array");
+        var inValueForSecondArray = in.nextInt();
+        System.out.println("Pls enter step of 2nd array");
+        var stepForSecondArray = in.nextInt();
+        System.out.println("Pls enter increment operator of 2nd array (+),(*),(-),(^) or (/)");
+        String operatorForSecondArray = in.next();
+        System.out.println("Pls enter operator to calculate the third array (+),(*),(-),(^) or (/)");
+        String GeneralOperator = in.next();
+        boolean x = false;
 
-    }
 
+        double[] array1 = new double[sizeOfArrays];
+        array1[0] = inValueForFirstArray;
+
+        for (int i = 1; i < sizeOfArrays; i++) {
+            switch (operatorForFirstArray) {
+                case "+":
+                    array1[i] = array1[i - 1] + stepForFirstArray;
```

Comment:
Can we write this in the next way?
```suggestion
                    array1[i] = array1[i--] + stepForFirstArray;
```